sql/_32_damson/cbrd_22856_Row_Value_Constructor/cases/Row_Value_Constructor_queryPlan.sql

```diff
@@ -17,7 +17,7 @@ select /*+ recompile */ * from rvc_tmp2 where (col1,col2) IN ((2,4),(1,2)) and (
 select /*+ recompile */ * from rvc_tmp2 where (col1,col2) IN ((2,4),(1,2)) and (col1,col2) in ((3,4),(2,3));
 select /*+ recompile */ * from rvc_tmp2 where (col1,col4) IN ((1,2),(2,2)) and col2 > 1;
 select /*+ recompile */ * from rvc_tmp2 where (col1,col3) IN ((2,4),(1,3)) and (col4,col5) in ((4,5),(2,3));
-select /*+ recompile */ * from rvc_tmp2 where (col1,col2) in (select 1,2 from rvc_tmp2);
+select /*+ recompile ordered */ * from rvc_tmp2 where (col1,col2) in (select 1,2 from rvc_tmp2);
```

Comment:
Don't you need to modify the answer file?

Reply:
yes, because this query is cross join.
but, the query optimizer is when execution the cross join, random select the outer or inner query.
so, 'shparkcubrid' is a request to modify this query.
(refer to https://circleci.com/gh/CUBRID/cubrid/17761#tests/containers/1)

modified the query always shows the same result.